docs/dev/stream/python.md

```diff
@@ -50,7 +53,12 @@ existing Java streaming APIs.
 There are two main constraints for using Jython:
 
 * The latest Python supported version is 2.7
-* It is not straightforward to use Python C extensions
+* It is not straightforward to use Python C extensions, which may prevent reuse of some libraries
+
+(For more information please see <https://wiki.python.org/jython/JythonFaq/GeneralInfo>.)
+
+One possible alternative for streaming that allows for native Python execution would be the Apache Beam
```

Comment:
add a link to the Apache Beam project, or maybe even a more specific python-related page.